apps/framework-cli/src/cli/routines.rs

```diff
@@ -384,6 +388,12 @@ async fn manage_leadership_lock(
                 e
             );
         }
+    } else if IS_RUNNING_LEADERSHIP_TASKS.load(Ordering::SeqCst) {
```

Comment:
Where / how does IS_RUNNING_LEADERSHIP_TASKS becomes true? It's initialized as false on line 132 and is set to false on line 396 but I don't see where it's going to be true. Thanks for explaining.

Reply:
This is correct and is an issue.  Updating PR now.